src/tasks/minions/woodcuttingActivity.ts

```diff
@@ -260,8 +260,8 @@ export const woodcuttingTask: MinionTask = {
 
 		// Add crystal shards for chopping teaks/mahogany in priff
 		if (forestry && priffUnlocked && resolveItems(['Teak logs', 'Mahogany logs']).includes(log.id)) {
-			// 15 Shards per hour
-			loot.add('Crystal shard', Math.floor((duration / Time.Hour) * 15));
+			// 1/40 chance of receiving a crystal shard
+			loot.add('Crystal shard', Math.round(quantity / 40));
```

Comment:
I could be wrong, but isnt this a 1/20 chance? did you do the math on this? 
you could also just do:
for (let i = 0; i < quantity; i++) {
 if (roll(40)) loot.add()
}

Reply:
The math is technically correct, but rounding is not the best way to do this now that you mention it, I was just lazy. Changed it to the loop.